GitIgnoreTest.testInclude fails on Windows

```
2023-03-15T10:41:51.8582653Z [ERROR] io.helidon.build.maven.enforcer.GitIgnoreTest.testInclude  Time elapsed: 0 s  <<< FAILURE!
2023-03-15T10:41:51.8583213Z java.lang.AssertionError: 
2023-03-15T10:41:51.8583781Z 
2023-03-15T10:41:51.8583973Z Expected: is <true>
2023-03-15T10:41:51.8584279Z      but: was <false>
2023-03-15T10:41:51.8584676Z 	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
2023-03-15T10:41:51.8585163Z 	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
2023-03-15T10:41:51.8585743Z 	at io.helidon.build.maven.enforcer.GitIgnoreTest.testInclude(GitIgnoreTest.java:156)
2023-03-15T10:41:51.8586336Z 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2023-03-15T10:41:51.8586935Z 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
2023-03-15T10:41:51.8587607Z 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2023-03-15T10:41:51.8588169Z 	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
2023-03-15T10:41:51.8592708Z 	at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:688)
2023-03-15T10:41:51.8593502Z 	at org.junit.jupiter.engine.execution.MethodInvocation.proceed(MethodInvocation.java:60)
2023-03-15T10:41:51.8595924Z 	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$ValidatingInvocation.proceed(InvocationInterceptorChain.java:131)
2023-03-15T10:41:51.8596714Z 	at org.junit.jupiter.engine.extension.TimeoutExtension.intercept(TimeoutExtension.java:149)
2023-03-15T10:41:51.8597417Z 	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestableMethod(TimeoutExtension.java:140)
2023-03-15T10:41:51.8598185Z 	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestMethod(TimeoutExtension.java:84)
2023-03-15T10:41:51.8600235Z 	at org.junit.jupiter.engine.execution.ExecutableInvoker$ReflectiveInterceptorCall.lambda$ofVoidMethod$0(ExecutableInvoker.java:115)
2023-03-15T10:41:51.8600942Z 	at org.junit.jupiter.engine.execution.ExecutableInvoker.lambda$invoke$0(ExecutableInvoker.java:105)
2023-03-15T10:41:51.8601820Z 	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$InterceptedInvocation.proceed(InvocationInterceptorChain.java:106)
2023-03-15T10:41:51.8609890Z 	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.proceed(InvocationInterceptorChain.java:64)
2023-03-15T10:41:51.8664728Z 	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.chainAndInvoke(InvocationInterceptorChain.java:45)
2023-03-15T10:41:51.8665759Z 	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.invoke(InvocationInterceptorChain.java:37)
2023-03-15T10:41:51.8666460Z 	at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:104)
2023-03-15T10:41:51.8667088Z 	at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:98)
2023-03-15T10:41:51.8994359Z 	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeTestMethod$6(TestMethodTestDescriptor.java:210)
2023-03-15T10:41:51.8998134Z 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
2023-03-15T10:41:51.9006230Z 	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeTestMethod(TestMethodTestDescriptor.java:206)
2023-03-15T10:41:51.9017043Z 	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:131)
2023-03-15T10:41:51.9017874Z 	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:65)
2023-03-15T10:41:51.9018618Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:151)
2023-03-15T10:41:51.9019330Z 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
2023-03-15T10:41:51.9020029Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
2023-03-15T10:41:51.9020667Z 	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
2023-03-15T10:41:51.9021295Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
2023-03-15T10:41:51.9022210Z 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
2023-03-15T10:41:51.9022997Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
2023-03-15T10:41:51.9023728Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
2023-03-15T10:41:51.9024308Z 	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
2023-03-15T10:41:51.9025101Z 	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
2023-03-15T10:41:51.9025961Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
2023-03-15T10:41:51.9026794Z 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
2023-03-15T10:41:51.9027490Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
2023-03-15T10:41:51.9028166Z 	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
2023-03-15T10:41:51.9028798Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
2023-03-15T10:41:51.9029498Z 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
2023-03-15T10:41:51.9030210Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
2023-03-15T10:41:51.9030895Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
2023-03-15T10:41:51.9031421Z 	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
2023-03-15T10:41:51.9032156Z 	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
2023-03-15T10:41:51.9033022Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
2023-03-15T10:41:51.9033725Z 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
2023-03-15T10:41:51.9034413Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
2023-03-15T10:41:51.9035035Z 	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
2023-03-15T10:41:51.9035665Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
2023-03-15T10:41:51.9036365Z 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
2023-03-15T10:41:51.9037105Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
2023-03-15T10:41:51.9037788Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
2023-03-15T10:41:51.9038740Z 	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:35)
2023-03-15T10:41:51.9039647Z 	at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
2023-03-15T10:41:51.9040419Z 	at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:54)
2023-03-15T10:41:51.9041148Z 	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:107)
2023-03-15T10:41:51.9041862Z 	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:88)
2023-03-15T10:41:51.9042587Z 	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.lambda$execute$0(EngineExecutionOrchestrator.java:54)
2023-03-15T10:41:51.9043468Z 	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.withInterceptedStreams(EngineExecutionOrchestrator.java:67)
2023-03-15T10:41:51.9044234Z 	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:52)
2023-03-15T10:41:51.9044904Z 	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:114)
2023-03-15T10:41:51.9045478Z 	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:86)
2023-03-15T10:41:51.9046127Z 	at org.junit.platform.launcher.core.DefaultLauncherSession$DelegatingLauncher.execute(DefaultLauncherSession.java:86)
2023-03-15T10:41:51.9046825Z 	at org.junit.platform.launcher.core.SessionPerRequestLauncher.execute(SessionPerRequestLauncher.java:53)
2023-03-15T10:41:51.9047518Z 	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.execute(JUnitPlatformProvider.java:188)
2023-03-15T10:41:51.9048252Z 	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invokeAllTests(JUnitPlatformProvider.java:154)
2023-03-15T10:41:51.9048973Z 	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invoke(JUnitPlatformProvider.java:128)
2023-03-15T10:41:51.9049645Z 	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
2023-03-15T10:41:51.9050241Z 	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
2023-03-15T10:41:51.9050783Z 	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
2023-03-15T10:41:51.9051307Z 	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
2023-03-15T10:41:51.9051638Z 
2023-03-15T10:41:51.9051964Z [INFO] Running io.helidon.build.maven.enforcer.nativeimage.NativeImageRuleTest
2023-03-15T10:41:51.9052846Z [INFO] Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.015 s - in io.helidon.build.maven.enforcer.nativeimage.NativeImageRuleTest
2023-03-15T10:41:52.0345196Z [INFO] 
2023-03-15T10:41:52.0346158Z [INFO] Results:
2023-03-15T10:41:52.0346628Z [INFO] 
2023-03-15T10:41:52.0347000Z [ERROR] Failures: 
2023-03-15T10:41:52.0347352Z [ERROR]   GitIgnoreTest.testInclude:156 
2023-03-15T10:41:52.0347694Z Expected: is <true>
2023-03-15T10:41:52.0347986Z      but: was <false>
```

--- a/maven-plugins/enforcer-maven-plugin/src/main/java/io/helidon/build/maven/enforcer/GitIgnore.java
+++ b/maven-plugins/enforcer-maven-plugin/src/main/java/io/helidon/build/maven/enforcer/GitIgnore.java
@@ -24,6 +24,8 @@ import java.util.regex.Pattern;
 
 import io.helidon.build.common.logging.Log;
 
+import static io.helidon.build.common.Strings.normalizePath;
+
 /**
  * Utility class for gitignore parsing.
  */
@@ -142,7 +144,7 @@ class GitIgnore implements FileMatcher {
 
     private boolean isParentExcluded(String pattern) {
         pattern = "/" + pattern;
-        Path parent = Path.of(pattern).getParent();
+        String parent = normalizePath(Path.of(pattern).getParent().toString());
         if (!parent.endsWith("/")) {
             pattern = parent + "/";
         }

